src/_gettsim/transfers/rente.py

```diff
@@ -19,8 +19,104 @@ def sum_ges_rente_priv_rente_m(priv_rente_m: float, ges_rente_m: float) -> float
     return out
 
 
-@policy_info(params_key_for_rounding="ges_rente")
-def ges_rente_vor_grundr_m(
+@policy_info(end_date="2020-12-31")
+def ges_rente_m(ges_rente_vor_grundr_m: float) -> float:
+    return ges_rente_vor_grundr_m
+
+
+@policy_info(
+    start_date="2021-01-01",
+    name_in_dag="ges_rente_m",
+    params_key_for_rounding="ges_rente",
+)
+def ges_rente_m_nach_grundr(
+    ges_rente_vor_grundr_m: float,
+    grundr_zuschlag_m: float,
+    rentner: bool,
+) -> float:
+    """Calculate total individual public pension including Grundrentenzuschlag. Is only
+    active after 2021 when Grundrente is in place.
+
+    Parameters
+    ----------
+    ges_rente_vor_grundr_m
+        See :func:`ges_rente_vor_grundr_m`.
+    grundr_zuschlag_m
+        See :func:`grundr_zuschlag_m`.
+    rentner
+        See basic input variable :ref:`rentner <rentner>`.
+
+    Returns
+    -------
+
+    """
+    # Return 0 if person not yet retired
+    out = ges_rente_vor_grundr_m + grundr_zuschlag_m if rentner else 0.0
+    return out
+
+
+@policy_info(
+    end_date="2022-12-31",
+    name_in_dag="ges_rente_vor_grundr_m",
+    params_key_for_rounding="ges_rente",
+)
+def ges_rente_vor_grundr_mit_hinzuverdienstgrenzen_m(
```

Comment:
Let's remove this level. `vor_grundrente` is confusing the hell out of me, because it could be meant in a temporal sense or a DAG-sequential sense.

Just move the age check into `bruttorente_mit_hinzuverdienstgrenzen`.